### PR TITLE
l10n: Improve style of sentence

### PR DESCRIPTION
**Pull request by Valdnet**
The text string has been shortened, which improves the style of the sentence.

Signed-off-by: Valdnet <47037905+Valdnet@users.noreply.github.com>

### CODE REVIEW

File: src/components/settings/AdminSettings.vue [hunk 1]
```diff
@@ -125,7 +125,7 @@
 					{{
 						t(
 							'mail',
-							'Any email that is marked as spam will not only be flagged as such but additionally sent to the anti spam service.'
+							'Any email that is marked as spam will additionally sent to the anti spam service.'
```
Comment (ChristophWurst) [inline src/components/settings/AdminSettings.vue:128]:
```suggestion
							'Any email that is marked as spam will be sent to the anti spam service.'
```

**Review by ChristophWurst — APPROVED** [review]
Please squash into one commit

File: src/components/settings/AdminSettings.vue (hunk 1 above)
Comment (Valdnet) [reply src/components/settings/AdminSettings.vue]: Thanks 😉. It sounds even better now.

**Comment by Valdnet** [comment]
> Please squash into one commit

I create commits via GitHub and I can't really squash commits as I know.

**Pull request merged**